db/migrate/20170706152142_add_category_option_id_to_categories.rb

```diff
@@ -0,0 +1,8 @@
+class AddCategoryOptionIdToCategories < ActiveRecord::Migration
+  def change
+    Category.all.each do |cat|
+      cat.category_option_id = CategoryOption.find_by(name: cat.name).id
+      cat.save
```

Comment:
It would be safer to do `save!` which will raise an exception and stop if something happens